packages/aws-cdk-lib/core/README.md

```diff
@@ -1663,10 +1663,15 @@ import { RemovalPolicys } from 'aws-cdk-lib';
 RemovalPolicys.of(scope).destroy();
 
 // Apply RETAIN policy only to specific resource types
-RemovalPolicys.of(scope).retain({
-  applyToResourceTypes: ['AWS::S3::Bucket', 'AWS::DynamoDB::Table'],
+RemovalPolicies.of(parent).retain({
+  applyToResourceTypes: [
+    'AWS::DynamoDB::Table',
+    bucket.cfnResourceType, // 'AWS::S3::Bucket'
+  ],
```

Comment:
Addition of an example of `(anyL1).CFN_RESOURCE_TYPE_NAME` might be helpful as well.

Reply:
Is this how it's supposed to be?
https://github.com/aws/aws-cdk/pull/32283/commits/5f6197c18c209440b83bfde5927adf643a71c468

Reply:
Yes, that's exactly what I said. Thanks.